.github/workflows/workflow-build-and-release-services.yml

```diff
@@ -0,0 +1,86 @@
+name: Release services
+
+on:
+  workflow_call:
+    inputs:
+      release-version:
+        description: 'Services release version'
+        type: string
+        required: false
+      snapshot-release:
+        description: 'Snapshot release?'
+        type: boolean
+        required: true
+        default: false
+      maven-release:
+        description: 'Release maven'
+        type: boolean
+        required: true
+        default: true
+      docker-release:
+        description: 'Release docker images'
+        type: boolean
+        required: true
+        default: true
+
+jobs:
+  release-services:
+    runs-on: ubuntu-latest
+    steps:
+      - name: PREP / Check out Git repository
+        uses: actions/checkout@v4
+      - name: PREP / Install Java and Maven
+        uses: actions/setup-java@v4
+        with:
+          java-version: '17'
+          distribution: 'corretto'
+      - name: PREP / Set up QEMU
+        if: inputs.docker-release == true
+        uses: docker/setup-qemu-action@v3
+      - name: PREP / Set up Docker Buildx
+        if: inputs.docker-release == true
+        uses: docker/setup-buildx-action@v3
+      - name: PREP / Login to DockerHub
+        if: inputs.docker-release == true
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKER_MIRAGON_USERNAME }}
+          password: ${{ secrets.DOCKER_MIRAGON_PASSWORD }}
+
+
+      - name: BUILD / Build with Maven
+        if: inputs.maven-release == false
+        run: |
+          chmod +x ./mvnw
+          ./mvnw --batch-mode clean install
+
+      - name: RELEASE / Remove SNAPSHOT
+        if: inputs.snapshot-release == false
+        run: |
+          chmod +x ./mvnw
+          ./mvnw versions:set --batch-mode -DremoveSnapshot -DprocessAllModules
+      - name: RELEASE / Build and Release Maven
+        if: inputs.maven-release == true
+        uses: samuelmeuli/action-maven-publish@v1
+        with:
+          gpg_private_key: ${{ secrets.pgp_private_key }}
+          gpg_passphrase: ${{ secrets.pgp_private_key_secret }}
+          nexus_username: ${{ secrets.nexus_username }}
+          nexus_password: ${{ secrets.nexus_password }}
```

Comment:
Should we use something else for maven releases?

Reply:
i would use another one